apps/charterafrica/src/payload/blocks/Consultation.js

```diff
@@ -0,0 +1,121 @@
+import {
+  mapPlaylistLinkToId,
+  mapVideoUrlToId,
+} from "../utils/mapPlaylistLinkToId";
+
+const Consultations = {
+  slug: "consultation",
```

Comment:
```suggestion
const ConsultationsMultimedia = {
  slug: "consultation-multimedia",
```

Since we'll have a separate `ConsultationDocuments` I assume.